ee/docker-ee/centos.md

```diff
@@ -64,28 +62,24 @@ apply:
   solid-state media (SSD) is recommended. Do not start Docker until properly
   configured per the [storage guide](/storage/storagedriver/device-mapper-driver/){: target="_blank" class="_" }.
 
-### Find your Docker Engine - Enterprise repo URL
+### Locating the URL for the Docker Engine - Enterprise Repo
 
-To install Docker Enterprise, you will need the URL of the Docker Enterprise repository associated with your trial or subscription:
+The product URL associated with your trial or subscription is required to install Docker Engine - Enterprise, which can be attained using the following procedure (note that these instructions apply to Docker on CentOS and for Docker on Linux, which includes access to Docker Engine - Enterprise for all Linux distributions):
```

Comment:
Could you say "...install Docker Engine - Enterprise. You can use the following procedure to get the required url. These instructions apply to ..."